packages/webdriverio/src/commands/browser/scroll.ts

```diff
@@ -34,6 +35,18 @@ export default function scroll (
         return log.warn('"scroll" command was called with no parameters, skipping execution')
     }
 
+    const { isAppium } = sessionEnvironmentDetector({
+        capabilities: this.capabilities,
+        requestedCapabilities: this.requestedCapabilities
+    })
+
+    if (isAppium) {
```

Comment:
We do have some flags attached to the browser object to determine the environment
```suggestion
    if (browser.isMobile) {
```